pkg/filesystem/local_directory_darwin.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 
 	"github.com/buildbarn/bb-storage/pkg/filesystem/path"
+	"golang.org/x/sys/unix"
```

Comment:
Imports are grouped by second level domains in the Buildbarn tree. Please move this import next to the google.golang.org ones.